pkg/bench/rttanalysis/validate_benchmark_data.go

```diff
@@ -77,7 +77,7 @@ func runBenchmarkExpectationTests(t *testing.T, r *Registry) {
 
 	var results resultSet
 	var wg sync.WaitGroup
-	concurrency := ((system.NumCPU()*4 - 1) / r.numNodes) + 1 // arbitrary
+	concurrency := ((system.NumCPU() - 1) / r.numNodes) + 1 // arbitrary
```

Comment:
Spoke with @ajwerner on slack about this.  Removing the `*4` makes the tests run much faster with larger (>4) rewrite iterations.